perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1916,6 +1943,25 @@ sub ban_user : Private {
     return 1;
 }
 
+sub update_alerts : Private {
+    my ($self, $c) = @_;
+
+    my $changes;
+    for my $alert ( $c->stash->{user}->alerts ) {
```

Comment:
This could use stashed alerts if they were put on stash on lookup. But then I guess you need to update that stash for any deleted ones...